test/setup_environment.sh

```diff
@@ -414,6 +419,36 @@ if [[ -n ${build_jade} ]]; then
     cd ..
 fi
 
+if [[ -n ${build_bitbox02} ]]; then
+    # Clone digital bitbox02 firmware if it doesn't exist, or update it if it does
+    if [ ! -d "bitbox02-firmware" ]; then
+        git clone --recursive https://github.com/BitBoxSwiss/bitbox02-firmware.git
+        cd bitbox02-firmware
+    else
+        cd bitbox02-firmware
+        git fetch
+
+        # Determine if we need to pull. From https://stackoverflow.com/a/3278427
+        UPSTREAM=${1:-'@{u}'}
+        LOCAL=$(git rev-parse @)
+        REMOTE=$(git rev-parse "$UPSTREAM")
+        BASE=$(git merge-base @ "$UPSTREAM")
+
+        if [ $LOCAL = $REMOTE ]; then
+            echo "Up-to-date"
+        elif [ $LOCAL = $BASE ]; then
+            git pull
+        fi
+    fi
+
+    # Build the simulator. This is cached, but it is also fast
+    CONTAINER_VERSION=$(cat .containerversion)
+    docker pull shiftcrypto/firmware_v2:$CONTAINER_VERSION
+    # The safe.directory config is so that git commands work. even though the repo folder mounted in
+    # Docker is owned by root, which can be different from the owner on the host.
+    docker run -i --rm --volume ./:/bb02 shiftcrypto/firmware_v2:$CONTAINER_VERSION bash -c "git config --global --add safe.directory /bb02 && cd /bb02 && make -j simulator"
```

Comment:
Got this error running `setup_environment.sh`:
```
+ docker run -i --rm --volume ./:/bb02 shiftcrypto/firmware_v2:42 bash -c 'git config --global --add safe.directory /bb02 && cd /bb02 && make -j simulator'
docker: Error response from daemon: create ./: "./" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path.
```

I think you could use `"$(pwd):/bb02"` instead of `./:/bb02`.

Reply:
This was not an issue in CI, but in local maybe that created problems. Anyways, I changed to `$(pwd)` instead of `.` as you suggested. This should solve the error.